ospfd/ospf_bfd.c

```diff
@@ -266,8 +266,12 @@ DEFUN (ip_ospf_bfd_prof,
 	struct ospf_if_params *params;
 	int idx_prof = 4;
 
-	ospf_interface_enable_bfd(ifp);
```

Comment:
Overall changes looks good to me.
 
But  before this change BFD is enabled implicitly when the BFD profile configured but  after this we are allowing to configure bfd profile only when bfd is set. 
Please check if any topotest cases specific to bfd profile are need to be modified.

Reply:
now test is update